src/main/java/ru/investbook/InvestbookApplication.java

```diff
@@ -21,36 +21,34 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.web.servlet.context.ServletWebServerInitializedEvent;
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.context.ApplicationContextException;
-import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
+
 @SpringBootApplication
 @Component
 @EnableCaching
 @RequiredArgsConstructor
 public class InvestbookApplication {
 
-    private final InvestbookProperties properties;
+    private static final LoadingPageServer loadingPageServer = new LoadingPageServer();
```

Comment:
`loadingPageServer` instance shouldn't exists after app start. Please, move final static variable to local context.